cmake/CMakeLists.txt

```diff
@@ -0,0 +1,286 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(usd-unity-sdk LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 11)
+
+set(USD_VERSION "v19.05" CACHE STRING "USD version (git tag)")
+
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+    set(boost_url "https://downloads.sourceforge.net/project/boost/boost/1.55.0/boost_1_55_0.tar.gz")
```

Comment:
2 different boost versions for: Windows/Unix

Reply:
That's just following along with the versions from the USD build script (https://github.com/PixarAnimationStudios/USD/blob/38c70fb869c01228f82fd844a8c420860b5131c1/build_scripts/build_usd.py#L519), but it would be nice to figure out why they're not moving everything to VFX CY2018 (at least).